src/Altinn.AccessManagement/Program.cs

```diff
@@ -269,9 +269,11 @@ void ConfigureServices(IServiceCollection services, IConfiguration config)
         options.AddPolicy(AuthzConstants.POLICY_STUDIO_DESIGNER, policy => policy.Requirements.Add(new ClaimAccessRequirement("urn:altinn:app", "studio.designer")));
         options.AddPolicy(AuthzConstants.ALTINNII_AUTHORIZATION, policy => policy.Requirements.Add(new ClaimAccessRequirement("urn:altinn:app", "sbl.authorization")));
         options.AddPolicy("PlatformAccess", policy => policy.Requirements.Add(new AccessTokenRequirement()));
+        options.AddPolicy(AuthzConstants.MPSystemResource_Write, policy => policy.Requirements.Add(new ScopeAccessRequirement(AuthzConstants.MPSystemResource_Write)));
```

Comment:
This ScopeAccessRequirement policy is just a placeholder until a solution for #219 is available

Reply:
removed